tools/README.md

```diff
@@ -125,6 +125,28 @@ Ex: _Run alphanet-8.1 with westend 9030 runtime_
 npm run launch --parachain alphanet-8.1 --relay westend-9030
 ```
 
+### Fast local build
+
+If you want to use your local binary for parachain or relay chain, you can reduce your compilation
+time by including only the native runtimes you need.
+For that you have to carefully check which runtimes you need, both on the moonbeam side and on the
+polkadot side.
+
+Here is the list of cargo aliases allowing you to compile only some native rutimes:
+
+| command | native runtimes |
+|-|-|
+| `cargo moonbase`  | `moonbase, westend, polkadot`  |
+| `cargo moonbase-rococo`  | `moonbase, rococo, westend, polkadot` |
+| `cargo moonriver` | `moonriver, polkadot` |
+| `cargo moonriver-rococo` | `moonriver, rococo, polkadot` |
+| `cargo moonriver-kusama` | `moonriver, kusama, polkadot` |
+| `cargo moonbeam` | `moonbeam, polkadot` |
+| `cargo moonbeam-rococo` | `moonbeam, rococo, polkadot` |
+
+* The `moonbase` mative runtime require `westend` native runtime to compile.
+* The `polkadot` is always included.
```

Comment:
Why that ?

Reply:
Because polkadot does not provide a conditional feature for the polkadot runtime itself, it is necessarily included.

Reply:
Maybe adding a comment in the readme:
* The `polkadot` is always included. (This is requirement from polkadot repo)